game-core/src/main/java/games/strategy/engine/data/Route.java

```diff
@@ -448,13 +448,13 @@ public static ResourceCollection getMovementFuelCostCharge(
   private Tuple<ResourceCollection, Boolean> getFuelCostsAndIfChargedFlatFuelCost(
       final Unit unit, final GameData data, final boolean ignoreFlat) {
     final ResourceCollection resources = new ResourceCollection(data);
-    boolean chargedFlatFuelCost = false;
     if (Matches.unitIsBeingTransported().test(unit)) {
-      return Tuple.of(resources, chargedFlatFuelCost);
+      return Tuple.of(resources, false);
```

Comment:
The more tuples I see, the worse they seem to get. No idea what kind of value this method actually returns

Reply:
Actually returns what the method name describes: fuel costs (ResourceCollection) and if charged flat fuel cost (boolean).

Reply:
No disagreement, but it's one more thing to remember. If you can't also see the method name at the same time, it's yet another layer to understand. Whenever you have to reference other code to understand the current line, it's a bit of a maintenance code-smell.